XLS-0033d-CFTs/README.md

```diff
@@ -514,35 +515,72 @@ We propose using the following format for CFT amounts::
 
 ```json
 "amount": {
-  "issuer": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
-  "cft_asset": "USD",
+  "cft_issuance_id": "00070C4495F14B0E44F78A264E41713C64B5F89242540EE255534400000000000000",
   "value": "1"
 }
 ```
 
-The idea behind this format is that it adds only one new subfield to the `amount` field, but still distinguishes between itself and Issued Currency amounts. Using the CFT ID directly would not allow us to easily find the underlying `CFTokenIssuance` object because we would still need the `issuer` to know where to look for it.
+Since the `AssetCode` is an optional field, the `CFTokenIssuanceID` will be used to uniquely identify the CFT during a Payment transaction.
+
+### 1.3.5 The **`CFTokenTrust`** Transaction
```

Comment:
The transaction has already prefixed with CFToken so I believe the confusion will be minimal. 

I believe "trust" is the right term here, since the holder still need to trust the issuer in order to redeem the tokens, so I don't really see this word being exclusive to trustlines. I'm open to renaming this transaction but right now I can't really think of better term that could replace it.

Reply:
Actually I changed my mind. I am looking to change it to `CFTokenAuthorize`, I do think this is a better naming, especially when there is allow-listing involved.

Reply:
For posterity, this change was made via 270356c03f8e76e4ce4de002a8732bb6f8f122d2